### ISSUE TITLE
pom

### ISSUE THREAD

**Issue by FavorMylikes** [OpenTSDB/opentsdb#1033]
<activeByDefault>@maven_profile_cassandra@</activeByDefault>

**Comment by manolama** [comment]
Yeah that's odd, which version of OpenTSDB were you building from please? Thanks.

**Comment by IDerr** [comment]
Hi, @FavorMylikes , is it possible to have the version you are trying to build
